packages/tooltip/README.md

```diff
@@ -0,0 +1,61 @@
+# @vaadin/tooltip
+
+A web component for creating tooltips.
+
+[![npm version](https://badgen.net/npm/v/@vaadin/tooltip)](https://www.npmjs.com/package/@vaadin/tooltip)
+[![Discord](https://img.shields.io/discord/732335336448852018?label=discord)](https://discord.gg/PHmkCKC)
+
+```html
+<vaadin-button id="confirm">Confirm</vaadin-button>
+<vaadin-tooltip text="Click to save changes" for="confirm"></vaadin-tooltip>
+```
+
+[<img src="https://raw.githubusercontent.com/vaadin/web-components/master/packages/tooltip/screenshot.png" width="200" alt="Screenshot of vaadin-tooltip">](https://vaadin.com/docs/latest/ds/components/tooltip)
+
+## Installation
+
+Install the component:
+
+```sh
+npm i @vaadin/tooltip
+```
+
+Once installed, import the component in your application:
+
+```js
+import '@vaadin/tooltip';
+```
+
+## Themes
+
+Vaadin components come with two built-in [themes](https://vaadin.com/docs/latest/ds/customization/using-themes), Lumo and Material.
```

Comment:
Where should this link? Maybe https://vaadin.com/docs/latest/styling?

Reply:
Yes, that's probably a good idea.

Reply:
Done.